src/ns_turn_defs.h

```diff
@@ -184,6 +184,7 @@ typedef uint32_t turn_time_t;
 #error WRONG BYTE_ORDER SETTING
 #endif
 
+// NOLINTBEGIN(clang-diagnostic-string-compare)
```

Comment:
Suppresses clang-tidy warning about comparing the address of constant string literals is implementation specific behavior.